<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelProgressionTable</class>
 <widget class="QWidget" name="PanelProgressionTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0">
     <item row="3" column="0">
      <widget class="QLabel" name="labelFinalValue">
       <property name="text">
        <string>Final value:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Table</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>218</width>
              <height>88</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <widget class="WidgetTableProgression" name="tableWidget">
               <property name="cursor" stdset="0">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Graph</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="WidgetChart" name="widgetChart">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelInitialValue">
       <property name="text">
        <string>Initial value:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="PanelPrimitiveValue" name="panelPrimitiveValueInitial" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="4">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>98</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0">
      <widget class="PanelPrimitiveValue" name="panelPrimitiveValueFinal" native="true">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>27</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="pushButtonReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="WidgetSliderProgression" name="widgetSliderProgression" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTableProgression</class>
   <extends>QTableWidget</extends>
   <header>widgettableprogression.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetChart</class>
   <extends>QFrame</extends>
   <header>widgetchart.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSliderProgression</class>
   <extends>QWidget</extends>
   <header>widgetsliderprogression.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
